aim2dat/strct/structure_importer.py

```diff
@@ -226,6 +226,97 @@ def import_from_mp(
         else:
             return self._import_from_odb("mp", formulas, {}, download_kwargs)
 
+    def import_from_mofxdb(
+        self,
+        mofid: str = None,
+        mofkey: str = None,
+        vf_min: float = None,
+        vf_max: float = None,
+        lcd_min: float = None,
+        lcd_max: float = None,
+        pld_min: float = None,
+        pld_max: float = None,
+        sa_m2g_min: float = None,
+        sa_m2g_max: float = None,
+        sa_m2cm3_min: float = None,
+        sa_m2cm3_max: float = None,
+        name: str = None,
+        database: str = None,
+        telemetry: bool = True,
+        pressure_unit: str = None,
+        loading_unit: str = None,
+        query_limit: int = 1000,
```

Comment:
I haven't used this database in practice, so I can only comment on the implementation and the expected user experience. Many of the arguments seem to be related to ranges. Have you thought about exposing only the ranges? If a user wants to understand the function and opens the documentation or uses the `?` operator, the amount of inputs might be quite overwhelming. For example: `pld_min` and `pld_max` could only be exposed as `pld` or `pld_range`, i.e. as tuples.

Reply:
Thanks for the suggestion, I applied it and you are 100% correct. This implementation made the code much more readable.